clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleDistributionPolicy.java

```diff
@@ -21,10 +21,11 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-/** Distribution policy for selecting zones in a regional MIG. */
+/** Distribution policy for selecting zones and target shape in a regional MIG. */
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class GoogleDistributionPolicy {
   List<String> zones;
+  String targetShape;
```

Comment:
Went with `String` over `enum` since it is likely additional target shapes will be added in the future. Didn't want to have an error when trying to cache a distribution policy created outside of Spinnaker with a target shape not (yet) supported by Spinnaker.